### DISCUSSION TITLE
Open Api has wrong json when generate json in browser

### DISCUSSION THREAD

**Discussion by UrKas16** [OAI/OpenAPI-Specification#4207] [General]
Hello! I have a problem with the Open api specification in when generating json in the browser via a link.
Version  Open Api UI  org.springdoc:springdoc-openapi-ui:jar:1.6.0

I have a description of the field in the yml file
![image](https://github.com/user-attachments/assets/c991433e-bb7e-49af-8f53-a4012817b01d)

When i start my app and jpen link localhost:8082/swagger-ui.html, i have this page:
![swagger page](https://github.com/user-attachments/assets/a7c2984a-922b-4642-afae-00ee3d9a1173)

When I click on the link "/v3/api-docs", I get a generated json file:
![json swagger](https://github.com/user-attachments/assets/86ef548d-4376-45f4-a859-6d4ea1070d50)

As can be seen from the description of the yml-file, the field "ids" has a type: "string " and a length: 255.
But when json is generated, the length for this field is 2147483647.

Help please. How to fix this problem?

**Comment by handrews** [comment]
HI @UrKas16 – I'm afraid this repository is for work on the OpenAPI Specification itself.  We cannot provide tooling support, so please raise this with your tooling vendor.

I will say that your input YAML looks fine, although the `maximum` keyword doesn't take effect with arrays.  It should be harmless- it's allowed, it just doesn't do anything because `maximum` affects numbers, not arrays.  But it's possible that some tool might get confused over it.

**Reply by UrKas16** [reply]
Thanks for the detailed answer!!
